src/WaCore.Crud.ListSample1/Views/Home/Index.cshtml

```diff
@@ -18,5 +18,17 @@
         <div>@Html.ActionLink("View example", "Index", "BookListData") </div>
 
     </div>
+
+    <div class="col-md-6">
+        <h2>Cars list</h2>
+        <h4>Simple CRUD functionality</h4>
+        <div>
+            Use base classes for Service (WacCrudService), Controller (WacCrudController), Filter (WacFilter), List-Viewmodel (WacListDataVm) and Edit-Viewmodel (WacEditVm).
```

Comment:
You don't have WacCrudController, WacListDataVm and WacEditVm.
And you forgot to mention WacListDataRepository.

Reply:
Corrected